packages/backend/src/tests/propertyBased/arbitraries/rbitraryWhere.ts

```diff
@@ -0,0 +1,80 @@
+import { fc } from '@fast-check/vitest';
+import { AllTablesRowsMap } from '../properties/executeQuery';
+import { arbitraryWhereValue } from './arbitraryWhereValue';
+import { Schema, Where, getTableWhereableColumns } from '@synthql/queries';
+import { AnyDb } from '../../../types';
+
+export function arbitraryWhere<DB>({
+    schema,
+    allTablesRowsMap,
+    tableName,
+    validWhere,
+}: {
+    schema: Schema<DB>;
+    allTablesRowsMap: AllTablesRowsMap;
+    tableName: string;
+    validWhere: boolean;
+}): fc.Arbitrary<Where<AnyDb, string>> {
+    return fc
+        .constantFrom(...getTableWhereableColumns(schema, tableName))
+        .chain((columnName) => {
+            // TODO: We should remove this check once we resolve the `timestampz` issue
```

Comment:
You should add a description of the timestampz issue, we will certainly forget it otherwise.

In general try to make comments self contained. This way you can understand them without the need for additional context.